src/node_file.cc

```diff
@@ -1003,24 +1003,10 @@ static void ExistsSync(const FunctionCallbackInfo<Value>& args) {
   THROW_IF_INSUFFICIENT_PERMISSIONS(
       env, permission::PermissionScope::kFileSystemRead, path.ToStringView());
 
-  uv_fs_t req;
-  auto make = OnScopeLeave([&req]() { uv_fs_req_cleanup(&req); });
   FS_SYNC_TRACE_BEGIN(access);
-  int err = uv_fs_access(nullptr, &req, path.out(), 0, nullptr);
+  bool exists = std::filesystem::exists(path.ToStringView());
```

Comment:
Is this safe? My understanding of the C++17 standard is that this overload _might_ throw an exception (which would cause the process to abort since we compile without exceptions enabled).

Reply:
oh yeah, forgot about that. Definitely need to make sure before it's merged that this is safe.

Reply:
I expected `-fno-exceptions` gcc flag to warn on these cases. apparently, it doesn't. thank you for the review. I've updated the implementation.